internal/config/config.go

```diff
@@ -232,7 +232,7 @@ func setClowderConfiguration(v *viper.Viper, clowderConfig *clowder.AppConfig) {
 	if clowderConfig.Database != nil && clowderConfig.Database.RdsCa != nil {
 		var err error
 		if rdsCertPath, err = clowderConfig.RdsCa(); err != nil {
-			log.Warn().Err(err).Msg("Cannot read RDS CA cert")
+			slog.Warn("Cannot read RDS CA cert", slog.Any("error", err))
```

Comment:
slog also takes pairs of string / any. `slog.Any("error", err)` and `"error", err` are analogous.

```suggestion
			slog.Warn("Cannot read RDS CA cert", "error", err)
```

The syntax is shorter. I wouldn't use it for complex logging calls. I'd prefer it for logger calls with a single error argument. What do you think, @frasertweedale and @avisiedo ?

Reply:
IMHO I'd rather the syntax `slog.Any(...)`, as if the key/pair are not balanced, it is detected in compile time, and would be consistent with more complex structured log messages.